build.zig

```diff
@@ -30,15 +19,6 @@ pub fn build(b: *std.Build) void {
     const zstd_mod = zstd_dep.module("zstd");
     const zstd_c_lib = zstd_dep.artifact("zstd");
 
-    const lib = b.addStaticLibrary(.{
```

Comment:
why did you remove this?

Reply:
It doesn't appear to serve any purpose. I don't see a single instance of any `export`s in the source code, and the actual outputted binary is functionally empty.